### PR TITLE
neat code syle adjustments and general refactoring

### PR DESCRIPTION
**Pull request by julpo99**

### CODE REVIEW

File: neat/controller_neat.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 
 # implements controller structure for player
-class player_controller(Controller):
+class PlayerController(Controller):
```
Comment (fr30) [inline neat/controller_neat.py:5]: How about something like "NeatController"?

File: neat/optimization_specialist_neat.py [hunk 2]
```diff
@@ -19,7 +18,7 @@
 # Change the enemy here, the winner will be saved in winner_neat_[en].pkl
 # Keep in mind that if you run this file, the winner will be overwritten
 
-#en = 1 # fitness = 95.03
+en = 1 # fitness = 95.03
```
Comment (fr30) [inline neat/optimization_specialist_neat.py:21]: how about elevate it to the constant on the top of the file and change the name to `ENEMY_IDX`?

File: jupyter/TSP_SOLUTION.py
Comment (fr30) [inline jupyter/TSP_SOLUTION.py:1]: Is it the same file as TSP_STUDENT.ipynb?

**Review by fr30 — CHANGES_REQUESTED** [review]
I had few remarks, we should merge it once they're fixed

File: neat/optimization_specialist_neat.py (hunk 2 above)
Comment (julpo99) [reply neat/optimization_specialist_neat.py:21]: sure, I will do this, so it's the same notation for both neat and deap

File: neat/controller_neat.py (hunk 1 above)
Comment (julpo99) [reply neat/controller_neat.py:5]: in task 2 we will have in the same file the PlayerController and the EnemyController. As they did in the file demo_controller. so I think we can keep the name PlayerController

File: jupyter/test.py [hunk 3]
```diff
@@ -1,10 +1,7 @@
-import time
```
[the remark this answers is not in the document]
Comment (julpo99) [reply jupyter/test.py:1]: okay I will

File: jupyter/TSP_SOLUTION.py
Comment (julpo99) [reply jupyter/TSP_SOLUTION.py:1]: no this is the solution for the TSP_STUDENT

File: jupyter/TSP_STUDENT.ipynb
[the remark this answers is not in the document]
Comment (julpo99) [reply jupyter/TSP_STUDENT.ipynb:1]: they gave it to us in uppercase and I left it as it is

File: neat/controller_neat.py (hunk 1 above)
Comment (fr30) [reply neat/controller_neat.py:5]: Current name is a bit confusing, it doesn't tell us anything about the implementation or what it does. Imagine you see it imported in some other part of the code - you don't know what it does just by the name alone. It should include some more information.

Comment (julpo99) [reply neat/controller_neat.py:5]: we can call it PlayerControllerNeat and then in the future we will call the enemy controller "EnemyControllerNeat". 
So we can do the same for Deap, by calling it "PlayerControllerDeap"
What do you think about that?

Comment (fr30) [reply neat/controller_neat.py]: Yeah, that would work. I feel like "NNControllerNeat" and "NNControllerDeap" give us the most information

Comment (julpo99) [reply neat/controller_neat.py]: Yes but if you look at the demo_controller that they gave us they have 2 one for the player and one for the enemy.
I changed it to PlayerControllerNeat. if it is fine you can merge

Comment (fr30) [reply neat/controller_neat.py]: Alright, that's fine. Change the name for neat controller and we're good

**Review by fr30 — APPROVED** [review]

**Pull request merged**